cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -92,6 +92,8 @@ share
     -- Shelley specific
     vrfKey              Text Maybe
     opCert              ByteString Maybe    sqltype=hash32type
+
+    invalidBefore       Word64 Maybe        sqltype=uinteger
```

Comment:
This should be in the tx table, along with the `invalidAfter`. It's an attribute of a tx, not a block.

Reply:
Yeah, brain fart.